test/viewmodels/SecuritiesRequestedStatementsViewModelSpec.scala

```diff
@@ -0,0 +1,192 @@
+        Some(pdfLink),
+        "rowId",
+        "dateCellId",
+        "linkCellId"
+      )
+
+      val result = SecuritiesRequestedStatementsViewModel.renderEoriHeading(row)
+
+      val expectedHtml = h2Component(
+        msg = "cf.account.details.previous-eori",
+        id = Some("requested-statements-eori-heading-0"),
+        classes = "govuk-heading-s govuk-!-margin-bottom-2"
+      ).body.trim
+
+      result.map(_.body.trim.replaceAll("\\s+", " ")) mustBe Some(expectedHtml.replaceAll("\\s+", " "))
+    }
+
+    "return None when EORI is not present" in {
+      val row = StatementRow(
+        0,
+        None,
+        "10 July 2023 to 20 July 2023",
+        Some(pdfLink),
+        "rowId",
+        "dateCellId",
+        "linkCellId"
+      )
+
+      val result = SecuritiesRequestedStatementsViewModel.renderEoriHeading(row)
+
+      result mustBe None
+    }
+  }
+
+  "renderPdfLink" should {
+    "render the correct PDF link" in {
+      val pdf = Some(pdfLink)
+      val result = SecuritiesRequestedStatementsViewModel.renderPdfLink(pdf)
+
+      val expectedHtml = spanLinkComponent(
+        msg = s"PDF (${pdfLink.formattedSize})",
+        url = pdfLink.downloadURL,
+        classes = "file-link govuk-link",
+        spanClass = Some("govuk-visually-hidden"),
+        spanMsg = Some("Download PDF")
+      ).body.trim
+
+      result.body.trim.replaceAll("\\s+", " ") mustBe expectedHtml.replaceAll("\\s+", " ")
+    }
+  }
+
+  "renderMissingDocumentsGuidance" should {
+    "generate guidance" in {
+      val result: HtmlFormat.Appendable = SecuritiesRequestedStatementsViewModel.renderMissingDocumentsGuidance
+
+      val document: Document = Jsoup.parse(result.body)
+
+      val heading: Elements = document.select("#missing-documents-guidance-heading")
+
+      heading.size() mustBe 1
+    }
+  }
+
+  "hasSecurityStatementsForEori" should {
+    "return true if there are requested statements" in {
+      val securityStatements = Seq(
+        SecurityStatementsForEori(
+          eoriHistory = EoriHistory("EORI456", Some(LocalDate.parse("2023-07-10")), Some(LocalDate.parse("2023-07-20"))),
+          currentStatements = Seq(requestedStatement),
+          requestedStatements = Seq(requestedStatement)
+        )
+      )
+
+      val result = SecuritiesRequestedStatementsViewModel.hasSecurityStatementsForEori(securityStatements)
+
+      result mustBe true
+    }
+
+    "return false if there are no requested statements" in {
+      val securityStatements = Seq(
+        SecurityStatementsForEori(
+          eoriHistory = EoriHistory("EORI456", Some(LocalDate.parse("2023-07-10")), Some(LocalDate.parse("2023-07-20"))),
+          currentStatements = Seq(requestedStatement),
+          requestedStatements = Seq.empty
+        )
+      )
+
+      val result = SecuritiesRequestedStatementsViewModel.hasSecurityStatementsForEori(securityStatements)
+
+      result mustBe false
+    }
+  }
+}
+
+trait Setup extends SpecBase {
```

Comment:
Either rename the Setup to SpecBaseWithSetup or SecuritiesRequestedStatementsViewModelSpec should extends SpecBase and use new Setup for in clause. Its to maintain the consistency (the way it been done at all other places and repos).